app/services/preserved_object_handler.rb

```diff
@@ -165,7 +165,7 @@ def version_comparison_results(db_object, version_symbol)
     elsif incoming_version > db_object.send(version_symbol)
       results << result_hash(ARG_VERSION_GREATER_THAN_DB_OBJECT, db_object.class.name)
```

Comment:
don't we want results.concat here?

Reply:
i think we still want to use `<<` for single elements, and `concat` for lists

Reply:
indeed:
```
m2dl-suntzu-01-mbpx:preservation_core_catalog suntzu$ bundle exec rails c
Running via Spring preloader in process 59610
Loading development environment (Rails 5.1.4)
[1] pry(main)> x = ['a']
=> ["a"]
[2] pry(main)> x.concat 'abc'
TypeError: no implicit conversion of String into Array
from (pry):2:in `concat'
[3] pry(main)> 
```